docs/contributrs-quick-start.rst

```diff
@@ -0,0 +1,915 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+*************************
+Contributor's Quick Guide
+*************************
+
+.. contents:: :local:
+
+
+Prerequisites
+#############
+
+1. Docker Community Edition
+2. Docker Compose
+3. pyenv (you can also use pyenv-virtualenv or virtualenvwrapper)
+
+
+Installing Prerequisites on Ubuntu
+###################################
+
+
+Docker Community Edition
+-----------------------------------
+
+
+1. Installing required packages for Docker and setting up docker repo
+
+.. code-block:: bash
+
+  $ sudo apt-get update
+
+  $ sudo apt-get install \
+      apt-transport-https \
+      ca-certificates \
+      curl \
+      gnupg-agent \
+      software-properties-common
+
+  $ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+
+  $ sudo add-apt-repository \
+     "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+     $(lsb_release -cs) \
+     stable"
+
+2. Install Docker
+
+.. code-block:: bash
+
+  $ sudo apt-get update
+  $ sudo apt-get install docker-ce docker-ce-cli containerd.io
+
+3. Creating group for docker and adding current user to it.
+
+.. code-block:: bash
+
+  $ sudo groupadd docker
+  $ sudo usermod -aG docker $USER
+
+Note : After adding user to docker group Logout and Login again for group membership re-evaluation.
+
+4. Test Docker installation
+
+.. code-block:: bash
+
+  $ docker run hello-world
+
+
+
+
+Docker Compose
+--------------------------
+
+1. Check the latest version of Docker Compose version |here| and Install
+
+.. |here| raw:: html
+
+   <a href="https://docs.docker.com/compose/install/#install-compose-
+   on-linux-systems">here</a>
+
+.. code-block:: bash
+
+  $ sudo curl -L "https://github.com/docker/compose/releases/download/1.27.4/docker-compose-$(uname -s)-$(uname -m)" \
```

Comment:
```suggestion
  $ COMPOSE_VERSION="$(curl -s https://api.github.com/repos/docker/compose/releases/latest | grep '"tag_name":' | cut -d '"' -f 4)"
  $ COMPOSE_URL="https://github.com/docker/compose/releases/download/${COMPOSE_VERSION}/docker-compose-$(uname -s)-$(uname -m)"
  $ sudo curl -L "${COMPOSE_URL}" \
```
Creating new variables makes it easier to edit the script, e.g. when you want to download different version.

Reply:
Nice, i will add this in next commit.

Reply:
I wonder if it would be cleaner to use `jq`, but it may require the user to preinstall it (which I think it's fine.)

```
curl -s https://api.github.com/repos/docker/compose/releases/latest | jq '.tag_name'` -r
```

Reply:
@xinbinhuang Does jq give us any more benefits apart from a minimally more readable code?  I would like to use jq if we were to perform a bit more complex operations. For now the standard linux tools are sufficient.

Reply:
I think my comment is only a nitpicking: The only benefit here is more readable code in my opinion. Other than that, they perform the same function. I agree that it's good for now